src/rust/engine/process_execution/src/remote.rs

```diff
@@ -622,6 +639,9 @@ impl CommandRunner {
             OperationOrStatus::Status(status)
           }
           _ => {
+            context
+              .workunit_store
+              .increment_counter(Metric::RemoteExecutionErrors, 1);
```

Comment:
`RpcFailure` won't increment this one... is that a lower level error, or should this increment move up between `Err(err) =>` and `match err {..}` ?

Reply:
The counter will be incremented in the code that follows that `match`. Note that the `RpcFailure` match arm just deconstructs the error and turns it into a `OperationOrStatus::Status` instance.